go-controller/pkg/ovn/pods.go

```diff
@@ -191,7 +193,9 @@ func (oc *Controller) deleteLogicalPort(pod *kapi.Pod) (err error) {
 		}
 	}
 	podNsName := ktypes.NamespacedName{Namespace: pod.Namespace, Name: pod.Name}
-	oc.deleteGWRoutesForPod(podNsName, portInfo.ips)
+	if err := oc.deleteGWRoutesForPod(podNsName, portInfo.ips); err != nil {
+		return fmt.Errorf("deleteGWRoutesForPod failed for %s: %w", podDesc, err)
```

Comment:
proper wording for "Delete GW Routes for pod: %q failed: %ws"

Reply:
done.